.gitignore

```diff
@@ -1 +1,11 @@
 build/
+.project
+/WebRoot/js/zimbra/
```

Comment:
I would suggest not to add js folders that are currently copied from zm-ajax into .gitignore. 

Also, ant clean removes these if required.